gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -715,3 +847,56 @@ def __init__(
 
   def expand(self, pvalue):
     return pvalue.pipeline | Read(self._source)
+
+
+class WriteToVcf(PTransform):
+  """A PTransform for writing to VCF files."""
+
+  def __init__(self,
+               file_path_prefix,
```

Comment:
considering that we dont provide the suffix option, then can this just be 'file_path'?

Reply:
Done.